source/voting.rst

```diff
@@ -0,0 +1,96 @@
+.. _voting:
+
+############
+Voting Items
+############
+
+Decisions about the future of the project are made through discussion with all
+members of the community. All non-sensitive project management discussion takes
+place on the project developers’ mailing list, the issue tracker, and in the
+project meetings mentioned above. Occasionally, sensitive discussion occurs on
+a private list, but the project defaults to open meetings.
+
+The yt-project uses a “consensus seeking” process for making large,
+project-changing decisions. The
+group tries to find a resolution that has no open objections among project
+members. At any point during the discussion, any project member can call for
+a vote, which will conclude two weeks from the call for the vote. Any vote must
+be backed by a YTEP. If no option can gather two thirds of the votes cast, the
+decision is escalated to the Steering Committee, which in turn will use
+consensus seeking with the fallback option of a simple majority vote if no
+consensus can be found within a month. 
+This is what we hereafter may refer to as “the decision making process”.
+
+Decisions (in addition to adding project members and Steering Committee
+membership as above) are made according to the following rules (+1 is
+equivalently an approval vote on a pull request):
+
+Minor Documentation Changes
+---------------------------
+
+Minor Documentation changes, such as typo fixes, or addition / correction of a
+sentence: Requires +1 by a project member, no -1 by a project member (lazy
+consensus), happens on the pull request page. Project members are
+expected to give “reasonable time” to others to give their opinion on the pull
+request if they’re not confident others would agree.
+
+Code Changes and Major Documentation Changes
+--------------------------------------------
+
+Code changes and major documentation changes require +1 by two project members,
+no -1 by a project member (lazy consensus), happens on the issue of
+pull-request page.
+
+Changes to the API Principles and Changes to Dependencies or Supported Versions
+-------------------------------------------------------------------------------
+
+Changes to the API principles and changes to dependencies or supported versions
+happen via a Enhancement proposals (YTEPs) and follows the decision-making
+process outlined above.
+
+Changes to the Project Governance 
+---------------------------------------
+
+Changes to the project governance model use the same decision process outlined
+above. 
```

Comment:
Outline above where? Can you make a sphinx hyperlink to where you're referring to here?

Reply:
I added hyperlinks to a couple of sections here, which should hopefully help with flow.